pyblazing/pyblazing/apiv2/hive.py

```diff
@@ -1,46 +1,60 @@
+from itertools import repeat
 
-from pyhive import hive
-from TCLIService.ttypes import TOperationState
-import numpy as np
 import cudf
-from itertools import repeat
-import pandas as pd
+
+# from pyhive import hive
```

Comment:
if we are not using this import can you just delete it

Reply:
ok.